cluster-classification/script.whizzml

```diff
@@ -0,0 +1,56 @@
+;; Given a cluster, creates a batch centroid and uses the output dataset
```

Comment:
You line length seem little long in general.  Emacs `alt-q` is helpful for formatting lines to > 80 chars.